compair/api/users.py

```diff
@@ -597,6 +601,132 @@ def get(self):
 
         return {"statuses": statuses}
 
+# /id/lti/users
+class UserLTIListAPI(Resource):
+    @login_required
+    def get(self, user_uuid):
+
+        user = User.get_by_uuid_or_404(user_uuid)
+
+        require(MANAGE, User,
+            title="User's Third Party Logins Unavailable",
```

Comment:
"User's LTI account links Unavailable"
"Sorry, your system role does not allow you to view LTI account links for this user."